pkgs/development/python-modules/nbclassic/default.nix

```diff
@@ -14,23 +14,13 @@ buildPythonPackage rec {
   version = "0.4.3";
   disabled = pythonOlder "3.6";
 
-  # tests only on github
```

Comment:
Are the tests now on pypi?

Reply:
yes there are now tests on pypi

Reply:
... and all of them pass (some with negligible warnings) :

```
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.9.13, pytest-7.1.2, pluggy-1.0.0
rootdir: /build/nbclassic-0.4.3, configfile: pyproject.toml, testpaths: tests
plugins: tornasync-0.6.0.post2, anyio-3.5.0
collected 3 items

tests/test_notebookapp.py ...                                            [100%]

=============================== warnings summary ===============================
tests/test_notebookapp.py::test_tree_handler
tests/test_notebookapp.py::test_notebook_handler
tests/test_notebookapp.py::test_terminal_handler
  /nix/store/0bks19g8dkazgkd0xfxjw6wp2sam5q17-python3.9-tornado-6.2/lib/python3.9/site-packages/tornado/ioloop.py:350: DeprecationWarning: make_current is deprecated; start the event loop first
    self.make_current()

tests/test_notebookapp.py::test_tree_handler
tests/test_notebookapp.py::test_notebook_handler
tests/test_notebookapp.py::test_terminal_handler
  /nix/store/7dz69gjb9pc2hs95wnvnawj50xxf4vh2-python3.9-pytest-tornasync-0.6.0.post2/lib/python3.9/site-packages/pytest_tornasync/plugin.py:65: DeprecationWarning: make_current is deprecated; start the event loop first
    loop.make_current()

tests/test_notebookapp.py::test_tree_handler
tests/test_notebookapp.py::test_notebook_handler
tests/test_notebookapp.py::test_terminal_handler
  /nix/store/7dz69gjb9pc2hs95wnvnawj50xxf4vh2-python3.9-pytest-tornasync-0.6.0.post2/lib/python3.9/site-packages/pytest_tornasync/plugin.py:67: DeprecationWarning: clear_current is deprecated
    loop.clear_current()

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================== 3 passed, 9 warnings in 1.40s =========================
Finished executing pytestCheckPhase
pytestcachePhase
pytestRemoveBytecodePhase
/nix/store/ynnqwl293rx8pzwibnz24x567vp0gx5d-python3.9-nbclassic-0.4.3
```